lib/galaxy/managers/jobs.py

```diff
@@ -416,6 +417,8 @@ def view_show_job(trans, job, full: bool) -> typing.Dict:
                 job_dict['user_email'] = None
 
             job_dict['job_metrics'] = summarize_job_metrics(trans, job)
+    tool = trans.app.toolbox.get_tool(job.tool_id, tool_version=job.tool_version)
+    job_dict.update(tool_name=tool.name)
```

Comment:
what is the alternative? Should I create an endpoint for the name?

Reply:
That seems like overkill, but I would like to avoid adding a wide variety of attributes to the job endpoint. Would it be enough to put this under the `full` view ?

Reply:
Ah, I see. Yeah, that's probably a good idea